packages/core/src/components/Accordion/story.tsx

```diff
@@ -12,20 +12,20 @@ export default {
 
 export function multipleItemsWithBorders() {
   return (
-    <Accordion bordered onChange={action('onChange')}>
-      <Item title="Item 1" onToggle={action('onToggle')}>
+    <Accordion bordered onChange={() => console.log('onChange')}>
```

Comment:
made this change throughout because it was causing annoying delays and freezes. the console.log still logs out to the Action panel, so it works!